packages/core-ethereum/crates/core-ethereum-misc/Cargo.toml

```diff
@@ -21,10 +21,14 @@ libp2p-identity = { workspace = true }
 thiserror = "1.0"
 wasm-bindgen = { workspace = true, optional = true }
 wasm-bindgen-futures = { version = "0.4.36", features = ["futures-core-03-stream"], optional = true }
+multiaddr = "0.18"
+ethers = { version = "2", default-features = false, features = ["ethers-solc"] }
+k256 = { version = "0.13.0", features = ["arithmetic"] }
```

Comment:
Is this needed ? Can we move any low-level cryptographic functionality to `core-crypto` pls ?

Reply:
forgot about that, totally true